pyproject.toml

```diff
@@ -19,9 +19,10 @@ postgres = "^3.0.0"
 python-decouple = "^3.3"
 uvicorn = "^0.11.5"
 geopandas = "^0.7.0"
-wget = "^3.2"
 pylint = "^2.5.3"
```

Comment:
does pylint need to be here as well as dev-dependencies?

Reply:
definitely not - my bad - it sneaked in 2 months ago!